preprocess/CheckEnvs.py

```diff
@@ -372,17 +372,20 @@ def CheckEnvs(args):
             '[WARNING] Current maximum contig length {} is much smaller than default chunk size {}, You may set a smaller chunk size by setting --chunk_size=$ for better parallelism.'.format(
                 max(contig_length_list), DEFAULT_CHUNK_SIZE)))
 
-    if is_bed_file_provided:
+    if is_bed_file_provided and default_chunk_num > -1:
```

Comment:
We have an issue where contigs with characters illegal in directory names were causing the analysis to crash here. This solves the issue for us, but it might be worth considering a fix that allows contigs to have back-slashes and other special characters in all cases